java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -298,9 +298,23 @@ open class KotlinUsesExtractor(
             }
         }
 
+    private fun extractParentExternalClassLater(d: IrDeclaration) {
+        val p = d.parent
+        when (p) {
+            is IrClass -> extractExternalClassLater(p)
+            is IrExternalPackageFragment -> {
+                // The parent is a (multi)file class. We don't need
+                // extract it separately.
```

Comment:
This comment is copied, but it doesn't make sense. Do we miss a `to` at the end of the first line?

Reply:
Yup, ta; fixed